client/src/chat.ts

```diff
@@ -13,6 +13,7 @@ import * as modals from "./modals";
 import ChatMessage from "./types/ChatMessage";
 
 // Constants
+const TYPED_HISTORY_MAX_LENGTH = 250;
```

Comment:
i guess this shouldn't be capitalized because it isn't exported (as per Airbnb)

Reply:
But it is a "constant" (not just a const).  I'm going to move it into a constants file :)